templates/download/server/download.html

```diff
@@ -1,130 +1,269 @@
+          {% include "download/shared/_verify-checksums.html" %}
+        {% endwith %}
+      </p>
+    </div>
+    <div class="col">
+      {% include 'shared/_server-download-newsletter.html' %}
+    </div>
+  </div>
+</section>
+
+{% include 'download/shared/_server_weekly_news.html' %}
+
+<section class="p-section">
+  <hr class="p-rule is-fixed-width" />
+  <div class="row--50-50">
+    <div class="col">
+      <h2>Ubuntu CLI cheatsheet</h2>
+      <div class="p-section--shallow">
+        <p>Learn how to use the command line efficiently and get started with DevOps — from basic file management to LXD virtualisation and Ubuntu Pro.</p>
+      </div>
+      <button class="p-button--positive" onclick="toggleModal()">Download the cheatsheet</button>
+    </div>
+    <div class="col">
+      <div class="p-section--shallow p-image-wrapper">
+        {{
+          image (
+          url="https://assets.ubuntu.com/v1/639c1f12-server-cli-cheatsheet.png",
+          alt="CLI cheat sheet",
+          width="1200",
+          height="849",
+          hi_def=True,
+          loading="auto"
+          ) | safe
+        }}
+      </div>
+    </div>
+  </div>
+</section>
+
+<section class="p-section">
+  <hr class="p-rule is-fixed-width" />
+  <div class="row">
+    <div class="col-3 col-medium-2">
+      <div class="p-section">
+        <h2 class="p-muted-heading">Resources</h2>
+      </div>
+    </div>
+    <div class="col-9">
+     <div class="row">
+      <div class="col-3 col-medium-2">
+        <h3 class="p-heading--5"><a href="/server/docs">Ubuntu Server documentation</a></h3>
+        <p>The Ubuntu Server documentation, curated by community and Ubuntu developers.</p>
+      </div>
+      <div class="col-3 col-medium-2">
+        <h3 class="p-heading--5"><a href="https://askubuntu.com/">Ask Ubuntu</a></h3>
+        <p>Stuck somewhere? Get your Ubuntu questions answered by power users.</p>
+      </div>
+      <div class="col-3 col-medium-2">
+        <h3 class="p-heading--5"><a href="https://discourse.ubuntu.com/">Community Discourse</a></h3>
+        <p>Discuss with Ubuntu developers and make your voice heard.</p>
+      </div>  
+     </div> 
+    </div>
+  </div>
+</section>
+
+{% include 'shared/_download-ubuntu-thank-you-pro-section.html' %}
+
+<section class="p-section">
+  <div class="p-section--shallow u-fixed-width">
+    <hr class="p-rule" />
+    <h2 class="p-muted-heading">More from Canonical</h2>
+  </div>
+  <div class="row">
+    <div class="col-3 col-medium-6">
       <div class="row">
-        <div class="col-6">
-          <h1 class="js-download-option">Thank you!</h1>
-          <p>Your download of Ubuntu Server {{ version }} should start in the background.</p>
+        <div class="col-6 col-medium-2">
+          <div class="p-section--shallow">
+            <h3>
+              <a href="https://maas.io/?_gl=1*1y3394w*_gcl_au*Mzg2MjE3ODk3LjE3MDI0NjY1NDY.&_ga=2.256850242.873815666.1709630805-805571671.1709630804">
```

Comment:
```suggestion
              <a href="https://maas.io/">
```
The added params aren't present on the doc

Reply:
It is the in the "learn more" link, just UX didn't add the link for logos...